tests/install_upgrade_operators/hco_enablement_golden_image_updates/test_update_hco_cr.py

```
@@ -41,24 +41,24 @@ def validate_template_dict(template_dict, resource_string):
     )
 
 
-@pytest.fixture(scope="class")
-def updated_hco_cr_custom_template_scope_class(
+@pytest.fixture()
```

Ruff (0.15.13)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)